web/src/app/schedules/ScheduleOverrideCreateDialog.js

```diff
@@ -82,6 +83,7 @@ ScheduleOverrideCreateDialog.propTypes = {
   scheduleID: p.string.isRequired,
   variant: p.oneOf(['add', 'remove', 'replace']).isRequired,
   onClose: p.func,
+  disableRemoveField: p.bool,
```

Comment:
I'm thinking this should be `removeUserReadOnly` based on function (even if we end up passing disabled to MUI)

That also follows the pattern in `CalendarSubscribeForm` which uses `scheduleReadOnly` to achieve the same purpose.